packages/dashboard/package.json

```diff
@@ -10,7 +10,7 @@
     "office-ui-fabric-react": ">=6.152.1 <7.0.0",
     "react-grid-layout-fabric": "^0.16.6",
     "react-resizable": "^1.7.5",
-    "react-transition-group":"^2.6.0",
+    "react-transition-group": "2.6.1",
```

Comment:
If we have issues with type mismatches can we just find a previous version of react-transition-group that references the previous versions? Alternatively, can we update our react-dom and react type references in our package.json?

Reply:
I tried v1 and still the same.
I tried updating the version, there is lot of other dependency issues due to that.
There is also work going on to have dashboards package as separate project and we will do version upgrade to latest in there.  This work will happen soon.  I will remove this when that work is done.

Reply:
Ok just wanted to confirm it wasn't a long term thing we'd have to maintain. Thanks for the clarification